eq-author/src/components/ToastContainer/index.js

```diff
@@ -2,17 +2,10 @@ import React from "react";
 import { connect } from "react-redux";
```

Comment:
Do we use toasts for anything more than undelete?

Reply:
Is that still required or can we remove it completely?

Reply:
🤷‍♂️ it's a UX change if we remove the Toast altogether, maybe @hamishtaplin has some thoughts?